src/web/test.rs

```diff
@@ -192,6 +195,46 @@ async fn basic() {
     }
 }
 
+#[actix_rt::test]
+async fn basic_old_ua() {
+    let adm = init_mock_adm(MOCK_RESPONSE1.to_owned());
+    let valid = ["acme", "los pollos hermanos"];
+    let mut settings = Settings {
+        adm_endpoint_url: adm.endpoint_url,
+        adm_settings: json!(adm_settings()).to_string(),
+        adm_has_legacy_image: Some(json!(valid).to_string()),
+        ..get_test_settings()
+    };
+    let mut app = init_app!(settings).await;
+
+    let req = test::TestRequest::get()
+        .uri("/v1/tiles")
+        .header(header::USER_AGENT, UA_90)
+        .to_request();
+    let resp = test::call_service(&mut app, req).await;
+    assert_eq!(resp.status(), StatusCode::OK);
+
+    let content_type = resp.headers().get(header::CONTENT_TYPE);
+    assert!(content_type.is_some());
+    assert_eq!(
+        content_type
+            .unwrap()
+            .to_str()
+            .expect("Couldn't parse Content-Type"),
+        "application/json"
+    );
+
+    let result: Value = test::read_body_json(resp).await;
+    let tiles = result["tiles"].as_array().expect("!tiles.is_array()");
+    assert!(tiles.len() > 1);
```

Comment:
Probably just forgot to modify this from `basic` -- we should assert_eq len() of 2 and that "Dunder Mifflin" was filtered out